src/main/java/core/mate/academy/service/BulldozerMachineProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerMachineProducer implements MachineProducer<Bulldozer> {
+    public List<Bulldozer> get() {
```

Comment:
FYI: 
it's better to use @Override annotation if you implementing interface method, that will make code more clear for reader

Reply:
Done